docs/user-documentation/metadata.md

```diff
@@ -135,25 +135,22 @@ seasonal value 2019-22 as defined in the EDTF specification):
 Example of the same EDTF dates displayed using little-endian format:
 ![Screenshot of dates displayed as '2019', 'November 2019', 'Summer 2019', and '2 February 2019 02:22:22Z'.](../assets/metadata_edtf_display.png)
 
-EDTF field values cannot include textual representations of dates, as shown below in this example of a valid EDTF value ('1943-05') and an invalid value ('1943 May') with the corresponding error message. Use the formatter configurations detailed further below to achieve textual display of dates.
-![Screenshot of both a valid ("1943-05") and an invalid ("1943 May") EDTF entry. Displays the error message "Could not parse the date 'May 1943' Years must be at least 4 characters long."](../assets/metadata_edtf_invalid.png)
+EDTF field values cannot include textual representations of dates, as shown below in this example of a valid EDTF value ('1943-05') and an invalid value ('May 1943') with the corresponding error message. Use the formatter configurations detailed further below to achieve textual display of dates.
+![Screenshot of both a valid ("1943-05") and an invalid ("May 1943") EDTF entry. Displays the error message "Could not parse the date 'May 1943' Years must be at least 4 characters long."](../assets/metadata_edtf_invalid.png)
 
 
 #### Configuration for the Default EDTF Widget
 
-This configuration can be set per field by clicking the **gear** icon next to any Default EDTF field at **Administration** \>\> **Structure** \>\> **Content types** \>\> **Repository Item** \>\> **Manage form display** (admin/structure/types/manage/islandora_object/form-display)
+This configuration can be set per field by clicking the **gear** icon next to any field defined with EDTF field type at **Administration** \>\> **Structure** \>\> **Content types** \>\> **Repository Item** \>\> **Manage form display** (admin/structure/types/manage/islandora_object/form-display)
```

Comment:
The phrase "Default EDTF field" raised questions for me. If I define a custom (i.e. not in Islandora Defaults) field with EDTF field type these instructions still work. The field type is called "EDTF", not "Default EDTF".